roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -2289,7 +2289,17 @@ RooDataSet *RooAbsPdf::generate(const RooArgSet &whatVars, const RooDataSet& pro
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooAbsPdf::translate(RooFit::Detail::CodeSquashContext &ctxManager) const
+{
+   // For now just return function/normalization integral.
+   std::string func = ctxManager.getResultName(servers()[0]->GetName());
+   std::string normIntg = ctxManager.getResultName(servers()[1]->GetName());
+   ctxManager.addResultName(GetName(), func + "/" + normIntg);
```

Comment:
```suggestion
   ctxManager.addResultName(this, func + "/" + normIntg);
```
Again, I would change the signature of `addResultName()` to take a `RooAbsArg const*` instead of the name directly, because this allows us to easily change how the keys are built from the objects later.